tests/functional/include/tests/utils.h

```diff
@@ -11,7 +11,7 @@
 
 #include "LogKit.h"
 
-namespace utils {
+namespace tests::utils {
```

Comment:
est-ce qu'on peut pas plutôt faire une extension de `leka::utils`?

```suggestion
namespace leka::utils {
```

puis dans les tests tu fais `using namespace leka`

Reply:
On retombe sur l'erreur avec ce changement

<img width="891" alt="Screenshot 2024-07-10 at 17 12 25" src="https://github.com/leka/LekaOS/assets/45776332/c23befcc-73ed-46a3-bec7-78f4cb523faa">